tests/test_api_jws.py

```diff
@@ -270,6 +270,16 @@ def test_verify_signature_with_no_secret(self, jws, payload):
 
         assert 'Signature verification' in str(exc.value)
 
+    def test_verify_signature_with_algo_none_throws_exception(self, jws, payload):
```

Comment:
I'd change this to say "test_verify_signature_with_no_algo_header_throws_exception..."

This will avoid confusion with `{'alg': 'none'}` vs `{'alg': None}`